racunaljka.js

```diff
@@ -0,0 +1,131 @@
+var writes = document.getElementById('screen');
+
+var addNumbers = document.querySelector('.allNumbers');
+var addEveryNumber = addNumbers.querySelectorAll('.number');
```

Comment:
Bad naming here, you are retrieving buttons, not actions (add..). Use normal intuitive names:
```
var numberButtons = document.querySelector('.number');
```